app/decorators/nav/link_builder.rb

```diff
@@ -0,0 +1,280 @@
+# frozen_string_literal: true
+
+# Defines nav menus and related helper methods.
+module Nav
+  class LinkBuilder < ApplicationDecorator
+
+    delegate_all
+
+    attr_accessor :context
+
+    def initialize()
+      context = {}
+    end
+
+    def nav_items
+      sample_period = Work::Period.new(community: h.current_community)
+      sample_job = Work::Job.new(period: sample_period)
+      sample_shift = Work::Shift.new(job: sample_job)
+      items = [
+        {
+          name: :people,
+          path: lens_path_if_present("users", index_path: h.users_path),
+          permitted: h.policy(User).index?,
+          icon: "users"
+        }, {
+          name: :meals,
+          path: lens_path_if_present("meals/meals", index_path: h.meals_path),
+          permitted: h.policy(Meals::Meal.new(community: h.current_community)).index?,
+          icon: "cutlery"
+        }, {
+          name: :work,
+          path: lens_path_if_present("work/shifts", index_path: h.work_shifts_path),
+          permitted: h.policy(sample_shift).index_wrapper?,
+          icon: "wrench"
+        }, {
+          name: :reservations,
+          path: lens_path_if_present("reservations", index_path: h.reservations_path),
+          permitted: h.policy(Reservations::Reservation.new(resource:
+                                                            Reservations::Resource.new(community: h.current_community))).index?,
+          icon: "book"
+        }, {
+          name: :wiki,
+          path: "/wiki",
+          permitted: h.policy(Wiki::Page.new(community: h.current_community)).show?,
+          icon: "info-circle"
+        }
+      ]
+      filter_and_set_active_nav_items(items, type: :main, active: @context[:section])
```

Comment:
Don't need the `@` here because you have `attr_accessor`. Omitting the @ is preferable.

Reply:
Same throughout.

Reply:
Got it.